src/storage/disk_manager.c

```diff
@@ -2699,8 +2660,10 @@ disk_cache_load_all_volumes (THREAD_ENTRY * thread_p)
   assert (disk_Cache != NULL);
   if (is_tran_server_with_remote_storage ())
     {
-      assert (disk_Page_server_perm_volume_count > 0);
-      for (VOLID volid = 0; volid < disk_Page_server_perm_volume_count; ++volid)
+      const DKNVOLS nvols_perm = xboot_find_number_permanent_volumes (thread_p);
+      assert (nvols_perm > 0);
+
+      for (VOLID volid = 0; volid < nvols_perm; ++volid)
```

Comment:
From what I heard in person, the disk_Cache on PTS doesn't know about permanent volumes. Why should we initialize it using disk_cache_load_volume()?

Reply:
Are you going to work on the following PRs right after this PR? If so, it's Ok. Or, I think at least we should leave a TODO comment.

Reply:
I mean not for all TODOs, just for TODO related to what is modified in this PR.

Reply:
I agree, I will leave an TODO comment for it.